airflow/providers/microsoft/azure/hooks/asb_admin_client.py

```diff
@@ -0,0 +1,83 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+from azure.servicebus.management import QueueProperties, ServiceBusAdministrationClient
+
+from airflow.providers.microsoft.azure.hooks.base_asb import BaseAzureServiceBusHook
+
+
+class AzureServiceBusAdminClientHook(BaseAzureServiceBusHook):
+    """
+    Interacts with Azure ServiceBus management client
+    and Use this client to create, update, list, and delete resources of a ServiceBus namespace.
+    it uses the same azure service bus client connection inherits from the base class
+    """
+
+    def __init__(self, *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+
+    def get_conn(self) -> ServiceBusAdministrationClient:
+        """Create and returns ServiceBusAdministration by using the connection string in connection details"""
```

Comment:
```suggestion
        """Create and returns ServiceBusAdministrationClient by using the connection string in connection details"""
```
Update for consistency with the return annotation.

Reply:
Fixed